apps/server/src/modules/health/repo/healthcheck.repo.mapper.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { HealthcheckRepoMapper } from './healthcheck.repo.mapper';
+import { HealthcheckEntity } from './entity';
+import { Healthcheck } from '../domain';
+
+describe(HealthcheckRepoMapper.name, () => {
+	describe(HealthcheckRepoMapper.mapHealthcheckEntityToDO.name, () => {
+		describe('when called with all the available fields', () => {
+			const setup = () => {
+				const testId = 'test_healthcheck_id';
+				const testUpdatedAt = new Date();
+				const testEntity = new HealthcheckEntity({ id: testId, updatedAt: testUpdatedAt });
+				const expectedDO = new Healthcheck(testId, testUpdatedAt);
+
+				return { testEntity, expectedDO };
+			};
+
+			it('should map to a valid object', () => {
+				const { testEntity, expectedDO } = setup();
+
+				const mappedDO = HealthcheckRepoMapper.mapHealthcheckEntityToDO(testEntity);
+
+				expect(mappedDO).toEqual(expectedDO);
+			});
+		});
+	});
+});
```

Comment:
```suggestion
import { HealthCheckRepoMapper } from './health-check.repo.mapper';
import { HealthCheckEntity } from './entity';
import { HealthCheck } from '../domain';

describe(HealthCheckRepoMapper.name, () => {
	describe(HealthCheckRepoMapper.mapHealthCheckEntityToDO.name, () => {
		describe('when called with all the available fields', () => {
			const setup = () => {
				const testId = 'test_healthcheck_id';
				const testUpdatedAt = new Date();
				const testEntity = new HealthCheckEntity({ id: testId, updatedAt: testUpdatedAt });
				const expectedDO = new HealthCheck(testId, testUpdatedAt);

				return { testEntity, expectedDO };
			};

			it('should map to a valid object', () => {
				const { testEntity, expectedDO } = setup();

				const mappedDO = HealthCheckRepoMapper.mapHealthCheckEntityToDO(testEntity);

				expect(mappedDO).toEqual(expectedDO);
			});
		});
	});
});
```

Reply:
Done in https://github.com/hpi-schul-cloud/schulcloud-server/pull/4191/commits/9f7cb45820e06668af950204e05f82983c112e54 .